src/base58.cpp

```diff
@@ -84,44 +90,47 @@ static const int8_t mapBase58[256] = {
     return true;
 }
 
+auto ConvertToLongs(const Span<const unsigned char>& input, int start) -> std::vector<int64_t> {
```

Comment:
pack to 64 bit  quickly (enabling native division for the groups), before converting to base58 to minimize the inner quadratic loop